form/form_themes.rst

```diff
@@ -26,16 +26,16 @@ do this, create a new template file that will store the new markup:
 
     .. code-block:: html+twig
```

Comment:
In fact, this line had to be indented less. Fixed in 044f60ac3fd9a1c32b37a5f709402db6411aa3bc.

Reply:
and the (final) fix in a8ea2d247ccd04ec5e69b3712832075954e47332